internal/cmd/commands/server/server.go

```diff
@@ -159,6 +160,27 @@ func (c *Command) Run(args []string) int {
 		globals.DefaultMaxRequestDuration = c.Config.DefaultMaxRequestDuration
 	}
 
+	// Perform controller configuration overrides for auth token settings
+	if c.Config.Controller != nil {
```

Comment:
Rather than put this code here, please instead update the configuration parsing (which then means any given HCL config will always work for this purpose even without having to go through the block here in the server command). The general way to do this is to have a raw interface{} value for the actual HCL to be decoded into, then call parseutil.ParseDurationSecond to convert what came in (which can be string or integer) into an integer number of seconds. Then you know that after config parsing the number will always either be zero or an integer number of seconds.

Reply:
Added this to `config.Parse()`, thanks!